front-end/src/App.tsx

```diff
@@ -29,9 +31,11 @@ function App() {
             <Route path="/" element={<Home />}></Route>
             <Route path="/search" element={<SearchPage />}></Route>
             <Route path="/player/:playerName" element={<Player />}></Route>
+            <Route path="/leagues/:leagueName" element={<LeaguePage />}></Route>
             <Route path="/404" element={<Error404 />}></Route>
             <Route path="/statexport" element={<StatExport />}></Route>
             <Route path="/leaderboard" element={<Leaderboards />}></Route>
+            <Route path="/leagues/:leagueName/:teamName" element={<TeamPage />}></Route>
```

Comment:
Should this be teamId? Or are we using a searchname for this as well?

Reply:
Ill probably update this when i get to implementing it but i think its going to actually be the abbreviation. That way people can just replace in the link with their own team name and go straight to the page

Reply:
Make sense.